Shaders/DisplayDepth.fx

```diff
@@ -8,251 +8,426 @@
+    ui_label = "Reversed";
+    ui_label_ja_jp = "深度バッファの奥行反転を修正";
+#if __RESHADE__ < 40500
+    ui_type = "combo";
+    ui_items = "RESHADE_DEPTH_INPUT_IS_REVERSED=0\0"
+               "RESHADE_DEPTH_INPUT_IS_REVERSED=1\0";
+#elif __RESHADE__ <= 50902
+    ui_type = "combo";
+    ui_items = "Off\0On\0";
+    ui_items_ja_jp = "オフ\0オン\0";
+#endif
+    ui_tooltip_ja_jp =
+        "画面効果が深度マップのとき、近くの形状が明るく、遠くの形状が暗い場合は変更して下さい。\n"
+        "また、法線マップで形が判別出来るが、深度マップが真っ暗に見えるという場合も、この設定の変更を試して下さい。"
+#if !ADDON_ADJUST_DEPTH
+        "\n\n"
+        "定義名は次の通りです。文字は完全に一致する必要があり、半角大文字の英字とアンダーバーを用いなければなりません。\n"
+        "RESHADE_DEPTH_INPUT_IS_REVERSED=値\n"
+        "定義値は次の通りです。オンの場合は1、オフの場合は0を指定して下さい。\n"
+        "RESHADE_DEPTH_INPUT_IS_REVERSED=1\n"
+        "RESHADE_DEPTH_INPUT_IS_REVERSED=0"
+#endif
+        ;
+> = RESHADE_DEPTH_INPUT_IS_REVERSED;
+
+#if __RESHADE__ <= 50902
+uniform int iUILogarithmic <
+#else
+uniform bool iUILogarithmic <
+#endif
+    ui_category = "Preview settings";
+    ui_label = "Logarithmic";
+    ui_label_ja_jp = "深度バッファを対数分布として扱うように修正";
+#if __RESHADE__ < 40500
+    ui_type = "combo";
+    ui_items = "RESHADE_DEPTH_INPUT_IS_LOGARITHMIC=0\0"
+               "RESHADE_DEPTH_INPUT_IS_LOGARITHMIC=1\0";
+#elif __RESHADE__ <= 50902
+    ui_type = "combo";
+    ui_items = "Off\0On\0";
+    ui_items_ja_jp = "オフ\0オン\0";
+#endif
+    ui_tooltip = "Change this setting if the displayed surface normals have stripes in them.";
+    ui_tooltip_ja_jp =
+        "画面効果に実際のゲーム画面と合致しない縞模様がある場合は変更して下さい。"
+#if !ADDON_ADJUST_DEPTH
+        "\n\n"
+        "定義名は次の通りです。文字は完全に一致する必要があり、半角大文字の英字とアンダーバーを用いなければなりません。\n"
+        "RESHADE_DEPTH_INPUT_IS_LOGARITHMIC=値\n"
+        "定義値は次の通りです。オンの場合は1、オフの場合は0を指定して下さい。\n"
+        "RESHADE_DEPTH_INPUT_IS_LOGARITHMIC=1\n"
+        "RESHADE_DEPTH_INPUT_IS_LOGARITHMIC=0"
+#endif
+        ;
+> = RESHADE_DEPTH_INPUT_IS_LOGARITHMIC;
+
+// -- Advanced options --
 
 uniform float2 fUIScale <
-	ui_category = "Advanced settings";
-	ui_type = "drag";
-	ui_label = "Scale (Preview)";
-	ui_tooltip = "Best use 'Present type'->'Depth map' and enable 'Offset' in the options below to set the scale.\n"
-	             "Use these values for:\nRESHADE_DEPTH_INPUT_X_SCALE=<left value>\nRESHADE_DEPTH_INPUT_Y_SCALE=<right value>\n"
-	             "\n"
-	             "If you know the right resolution of the games depth buffer then this scale value is simply the ratio\n"
-	             "between the correct resolution and the resolution Reshade thinks it is.\n"
-	             "For example:\n"
-	             "If it thinks the resolution is 1920 x 1080, but it's really 1280 x 720 then the right scale is (1.5 , 1.5)\n"
-	             "because 1920 / 1280 is 1.5 and 1080 / 720 is also 1.5, so 1.5 is the right scale for both the x and the y";
-	ui_min = 0.0; ui_max = 2.0;
-	ui_step = 0.001;
+    ui_category = "Preview settings";
+    ui_label = "Scale";
+    ui_label_ja_jp = "拡大率";
+    ui_type = "drag";
+    ui_text_ja_jp =
```

Comment:
Could add the text from `Advanced_help` here as a default again (since there already is a Japanese variant with similar text):
```c++
    ui_text =
        "\nThe following settings also need to be set using \"Edit global preprocessor definitions\" above in order to take effect.\n"
        "You can preview how they will affect the Depth map using the controls below.\n\n"
        "It is rarely necessary to change these though, as their defaults fit almost all games.";
```